recipes/AISHELL-1/NAR-ASR/NAR_BERT_ASR/prepare.py

```diff
@@ -0,0 +1,126 @@
+import os
```

Comment:
We already have a preparation script for AISHELL, could you please let us know why you need a new one ? Because this one doesn't look like a symlink.